Sources/Classes/RSServerConfigSource.m

```diff
@@ -23,4 +24,26 @@ - (void)addDestination:(RSServerDestination *)destination {
     [self.destinations addObject:destination];
 }
 
+- (NSString *) getDataResidencyUrl:(RSDataResidencyServer) residency {
+    NSArray * residenceDataPlanes;
+    switch(residency) {
+        case EU:
+            residenceDataPlanes = [self.dataPlanes objectForKey:@"EU"];
```

Comment:
Why break is not here? I don't understand the logic here. Can we have a call?

Reply:
Added a comment to explain the logic here